util/ci.sh

```diff
@@ -42,7 +38,7 @@ case $1 in
       exec rake test
     else
       cd bundler
-      exec rake spec:travis -t
+      exec rake spec -t
```

Comment:
This runs only the standard bundler tests, without "sudo" tests and "realworld" tests. Is this to make what's run by rubygems "lighter", or is there any problem with those tests? In case we just want to make this run faster, maybe is better to run only "realworld" tests? It would definitely be way faster, but I'm not sure it's thorough enough.

Reply:
We should add sudo and realworld to GitHub Actions. But I'm not sure how to handle them on Actions yet.